xapian-core/api/positioniterator.cc

```diff
@@ -76,6 +78,20 @@ PositionIterator::operator=(const PositionIterator & o)
     RETURN(*this);
 }
 
+PositionIterator::PositionIterator(PositionIterator && o)
+    : internal(o.internal)
+{
+    o.internal = nullptr;
+}
+
+PositionIterator &
+PositionIterator::operator=(PositionIterator && o)
+{
+    if (this != &o)
+	swap(internal, o.internal);
+    RETURN(*this);
```

Comment:
This should just be `return *this;` - only use `RETURN` if there's debug logging of the function/method call (i.e.  `LOGCALL()` or similar).

(These methods aren't really interesting to log, so I think it's fine not to have logging for them.)

Reply:
Fixed